tests/std/tests/P0645R10_text_formatting_legacy_text_encoding/env.lst

```diff
@@ -1,11 +1,9 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-# This is `concepts_matrix.lst` with `/execution-charset:.932` added.
+# This is `concepts_latest_matrix.lst` with `/execution-charset:.932` added.
 # clang is excluded since it doesn't support non-UTF-8 execution charsets.
 
-RUNALL_INCLUDE ..\prefix.lst
-RUNALL_CROSSLIST

```

Comment:
Why is `prefix.lst` being dropped?

Reply:
This is a straight-up bug. Reverified against `concepts_latest_matrix`.